libsplinter/src/biome/rest_api/config.rs

```diff
@@ -52,6 +59,7 @@ impl BiomeRestConfig {
 pub struct BiomeRestConfigBuilder {
     issuer: Option<String>,
     access_token_duration: Option<Duration>,
+    refresh_token_duration: Option<Duration>,
```

Comment:
Should this have a feature-guard, same with other instances of the `refresh_token_duration` within this file ?

Reply:
Guarded by feature